### PR TITLE
Filings Amendment Status

### PR DESCRIPTION
**Pull request by xtine**
Update Filings table with version and amendment information. https://github.com/18F/openFEC-web-app/issues/1738

screen shot 2017-01-03 at 3 00 09 pm

Relies on https://github.com/18F/fec-style/pull/599

### CODE REVIEW

**Comment by xtine** [comment]
@jontours: I need some logic help on the Original/Amendment <Version Number> description. I'm still getting confused because there are some filings that have `is_amended: false` but also has `amendment_indicator: A` and an `amendment_chain`, so is that Original or an Amendment? Also not totally sure how to get amendment version number based on `amendment_chain` and recent/previous file numbers.

**Comment by xtine** [comment]
@noahmanger: I've updated the amendment versioning logic that should be correct for all e-filings, but the data might not be totally correct for paper. Should I leave out the document versions for paper filings until the data is cleaned/verified?

**Comment by noahmanger** [comment]
@xtine so for paper, what would it show?

**Comment by xtine** [comment]
@noahmanger: it would just have the document name without original/amendment next to it like how it currently is.

**Comment by jontours** [comment]
Some of the paper filings are messy, so messy that not only can we not tell if it's an original or amended filing (without looking at the PDF), but we have not amendment chain for them.  After talking to @xtine we thought it may best to keep them as is until we can clean the data (we being us and the FEC).

**Comment by jontours** [comment]
Annddd it's even messier than that it seems, after digging around in the data some there are `e_file` rows that have null amendment indicators.  So @xtine maybe it's best to just not provide any of the new styling info if `amendment_indicator` is null.

**Comment by xtine** [comment]
I've updated it to only show e-filings and hiding any `null` values to avoid showing misleading data. It shows less data but I'd rather only show accurate data than more potentially incorrect data.

**Comment by noahmanger** [comment]
Are you hiding only the amendment status stuff or hiding the records entirely?

**Comment by xtine** [comment]
Just the amendment status stuff.

File: static/js/modules/columns.js [hunk 1]
```diff
@@ -268,20 +281,39 @@ var filings = {
...
+      if(row.means_filed !== 'e-file'|| row.amendment_indicator === null) {
+        show_version = false;
+      }
+
+      if (row.amendment_indicator === 'N') {
+        is_original = true;
+      }
+
+      if (row.amendment_chain) {
+        amendment_num = row.amendment_chain.length;
```
Comment (noahmanger) [inline static/js/modules/columns.js]: I think this should be `row.amendment_chain.length - 1`? The numbers seem like they're inflated by 1.

File: static/js/modules/columns.js [hunk 2]
```diff
@@ -29,6 +29,19 @@ var amendmentIndicatorColumn = {
   },
 };
 
+var versionColumn = {
+  data: 'most_recent',
+  className: 'hide-panel hide-efiling column--med min-desktop',
+  render: function(data) {
+    if (data === true) {
+      return '<div class="icon-circle--check-outline--inline--left"></div>Most recent version';
```
Comment (noahmanger) [inline static/js/modules/columns.js]: Small thing, but I see that elsewhere we usually do `<i>` for icons instead of `<div>`

**Comment by noahmanger** [comment]
This is great.

**Pull request merged**